src/operator/elemwise_op_common.h

```diff
@@ -128,29 +128,33 @@ inline bool ElemwiseAttr(const nnvm::NodeAttrs& attrs,
   if (n_out != -1)
     out_size = static_cast<size_t>(n_out);
 
-  auto deduce = [&](std::vector<AttrType> *vec, size_t size, const char *name) {
```

Comment:
Can we revert this change? The common practice is that if we change the elements in the vector, we will use pointer instead of const reference.

Reply:
Right. We are not changing the vector in this function if you look closely.